kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/OffsetFinder.java

```diff
@@ -75,7 +75,7 @@ public void findMessages(final long timestamp, AsyncCallbacks.FindEntryCallback
             }, this, callback);
         } else {
             if (log.isDebugEnabled()) {
-                log.debug("[{}][{}] Ignore message position find scheduled task, last find is still running");
+                log.debug("Ignore message position find scheduled task, last find is still running");
```

Comment:
also here, log the information of managedLedger.getName() and timestamp is useful